misk/src/test/resources/ssl/client.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Please update the README.md in this directory with the steps used to build these files, so that we know how to reconstruct them in case of expiration etc.